smartcontract/SwanFaucet.sol

```diff
@@ -16,13 +18,15 @@ contract SwanFaucet is OwnableUpgradeable {
     
     mapping(address => uint256) nextAccessTime;
 
+    mapping(address => bool) isAdmin;
+
     function initialize(address _tokenInstance) public initializer {
         require(_tokenInstance != address(0));
         tokenInstance = ERC20(_tokenInstance);
     }
 
     function requestTokens() public {
```

Comment:
I think we can remove this one.

Reply:
`mapping(address => bool) isAdmin;` i need this to keep track of admins